src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/xlf/MicrosoftNetCoreAnalyzersResources.cs.xlf

```diff
@@ -67,6 +67,21 @@
         <target state="translated">Vyhněte se konstantním polím jako argumentům</target>
         <note />
       </trans-unit>
+      <trans-unit id="AvoidSingleUseOfLocalJsonSerializerOptionsDescription">
+        <source>Avoid single-use of local 'JsonSerializerOptions' as it is suboptimal, use a singleton instance instead to improve performance.</source>
```

Comment:
"suboptimal" is probably an understatement, I would use somewhat stronger language along the lines of 

> Single-use 'JsonSerializerOptions' instances can substantially degrade the performance of your applicaton, consider using a singleton instead.

@stephentoub thoughts?

Reply:
Or just:
"Avoid creating a new `JsonSerializerOptions` instance for every serialization operation. Cache and reuse instances instead."